js/src/main/scala/kazari/KazariPlugin.scala

```diff
@@ -0,0 +1,94 @@
+package kazari
+
+import org.scalajs.dom
+import org.scalajs.dom._
+
+import scala.scalajs.js.JSApp
+import scala.scalajs.js.annotation.JSExport
+import dom.ext.PimpedNodeList
+import kazari.model.EvaluatorConfig
+import org.scalaexercises.evaluator.Dependency
+import org.scalaexercises.evaluator.{EvalResponse, EvaluatorClient}
+import org.scalaexercises.evaluator.EvaluatorClient._
+import org.scalaexercises.evaluator.implicits._
+import org.scalaexercises.evaluator.EvaluatorResponses.EvaluationResponse
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+@JSExport
+object KazariPlugin extends JSApp with DOMHelper {
+  val codeExcludeClass = "code-exclude"
+  lazy val codeSnippets = document.querySelectorAll(s"code.language-scala:not(.$codeExcludeClass)")
+  val dependenciesMetaName = "scala-dependencies"
```

Comment:
I would suggest changing these meta-tags for `evaluator-dependencies` and `evaluator-resolvers`. What do you think?